client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -224,6 +251,22 @@ const getAddressErrors = ( addressData, countriesData ) => {
 		}
 	}
 
+	if ( shouldValidatePhone ) {
+		// EasyPost requires an origin phone number for international shipments.
+		// This validation ensures that EasyPost will accept it, even though the phone may be invalid.
+		if ( ! phone ) {
+			errors.phone = translate( 'Origin phone number is required for international shipments' );
```

Comment:
Minor comment - but the phone number is required when customs form is required - since it may include US territories, etc.

Maybe something like:

"Please enter a phone number for your origin address. It's required because this shipment requires a customs form."

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/24974/commits/38e4dc095a548caa4328f797ea722c3dc7297a69